AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -133,12 +134,20 @@ - (void)enqueueItem:(id<MSLog>)item {
     // If sender is nil, there is nothing to do at this point.
     if (shouldFilter) {
       MSLogDebug([MSAppCenter logTag], @"Log of type '%@' was filtered out by delegate(s)", item.type);
+      [self enumerateDelegatesForSelector:@selector(channel:didCompleteEnqueueingLog:withInternalId:)
```

Comment:
Not a merge blocker but we need to think about not repeating this in many places in the method.